doc/Known_issues/Known_issues.md

```diff
@@ -4,33 +4,62 @@ Below is a list of known issues with workarounds.
 
 ---
 
-### IP fragmentation doesn't work
+### DNS doesn't work in my networks
+
+For DNS to work in containers, DHCP has to be turned on in Contrail network via WebUI.
+Please refer to this bug: https://bugs.launchpad.net/opencontrail/+bug/1535856
+
+---
+
+### I can't get IP fragmentation to work
 
 On Windows Server 2016, NAT network on compute node must be disabled in order for IP packet fragmentation to work.
 
+To do that, use the following commands:
+```
+Get-NetNat | Remove-NetNat                      # Remove existing NAT networks
+Stop-Service winnat                             # Stop WinNAT service
```

Comment:
I am not sure what are those doing, but all scripts turning off nat execute also:
Get-ContainerNetwork | Remove-ContainerNetwork -ErrorAction SilentlyContinue -Force
Get-ContainerNetwork | Remove-ContainerNetwork -Force

Reply:
Hmm good point, I'll add it